app/model/Model.scala

```diff
@@ -79,9 +79,9 @@ case class ConfirmPage(title: Option[String] = None,
                        confirmChangeText: Option[String] = None
                       )
 
-case class ResolvedLookupPage(p: LookupPage, isukMode: Boolean) {
-  val title: String = p.title.getOrElse(LOOKUP_PAGE_TITLE)
-  val heading: String = p.heading.getOrElse(LOOKUP_PAGE_HEADING)
+case class ResolvedLookupPage(p: LookupPage, isUkMode: Boolean) {
+  val title: String = p.title.getOrElse(LOOKUP_PAGE_TITLE).replaceAll("\\$UK", if (isUkMode) "UK" else "")
```

Comment:
You are using $UK as the place holder to find and replace it with either "UK" or empty based on the isUkMode flag.

It is not very clean and could be confusing as many Scala uses $ with in strings quotes for string interpolation and for substitute variable value.

Better approach would be to inject isUkMode flag to **JourneyConfigDefaults** trait and both **EnglishConstants** and **WelshConstants** needs to be case class in stead of objects.